core/server/controllers/frontend/channel-config.js

```diff
@@ -14,13 +13,13 @@ channelConfig = function channelConfig() {
             name: 'tag',
             route: '/' + config.routeKeywords.tag + '/:slug/',
             postOptions: {
-                filter: labs.isSet('hiddenTags') ? 'tags:\'%s\'+tags.hidden:false' : 'tags:\'%s\''
+                filter: 'tags:\'%s\''
```

Comment:
I have a slightly different, and I _think_ simpler solution for this on my branch: https://github.com/ErisDS/Ghost/commit/7dff6618fe415c03f94679c8308e8580e7a55334#diff-d23ab2c11f473dd33cad0f93fb92da6cR17